modules/cms/classes/Theme.php

```diff
@@ -148,8 +148,12 @@ public function isActiveTheme()
     public static function getActiveThemeCode()
     {
         $activeTheme = Config::get('cms.activeTheme');
+        $themes = static::all();
+        $havingMoreThemes = count($themes) > 1;
+        $themeHasChanged = !empty($themes[0]) && $themes[0]->dirName !== $activeTheme;
+        $checkDatabase = $havingMoreThemes || $themeHasChanged;
 
-        if (App::hasDatabase()) {
+        if ($checkDatabase && App::hasDatabase()) {
```

Comment:
@vojtasvoboda Following the above change, we just need to check `$hasMultipleThemes` at this point:
```suggestion
        if ($hasMultipleThemes && App::hasDatabase()) {
```

Reply:
@vojtasvoboda Please ignore this for now. I'm going to make a different suggestion in the PR.